docs/API.md

```diff
@@ -493,4 +494,18 @@ Parameters:
   * uid : user id
   * type : type of reminder
 
+## .createEmailBounce(body) ##
+
+Record when an email bounce has occurred.
+
+Parameters:
+
+* body: (object)
+  * email: A string of the email address that bounced
+  * bounceType: The bounce type ([`'Permanent'` or `'Transient'`])
+  * bounceSubType: The bounce sub type string
+  * template: The string name of the template that was used for the sent email
+  * lang: The language of the sent email
```

Comment:
I wonder whether we need to pay the cost of storing `template` and `tag`, or whether we should just log them for aggregate metrics purposes, but not store them in the database.  How do you see them being used?